store/src/java/com/zimbra/cs/service/account/ModifyPrefs.java

```diff
@@ -185,6 +200,45 @@ public static void sendEmailVerificationLink(Account authAccount, Account ownerA
         }
     }
 
+    private static String getImageBase64(ZimbraSoapContext zsc, String attachId)
```

Comment:
Can you make it public static and move it to a new class FileUploadServletUtil in package com.zimbra.cs.service.util

Reply:
Done